packages/vite-plugin-svelte/tsconfig.json

```diff
@@ -3,8 +3,8 @@
 	"compilerOptions": {
 		"noEmit": true,
 		"target": "ES2020",
```

Comment:
can we update target here?

Reply:
You mean something higher like "es2022"? It doesn't support "node*" strings